### DISCUSSION TITLE
Mangrove Carbon and Policy Widget

### DISCUSSION THREAD

**Discussion by tamaramegan** [Vizzuality/mangrove-atlas#285] [General]
Hello! 
I am opening this Discussion bout the Mangrove Carbon and Policy Widget. Here is [the presentation](https://docs.google.com/presentation/d/1dKieOCHcbSxNHBekKN9escwsTxYn8JCW/edit#slide=id.p1) we showed yesterday with the ideas we are proposing for this section, so you can share it and review it internally.

I have also added a slide with some comments about the data table you shared and some changes that would be very helpful fro the next round. I will write them here again for reference:
- Adding units to the column names (eg: FREL_CO2e_Mt_y-1, NDC_CO2e_mt_y-1, ha, etc)
- Splitting the values and errors into separate columns (eg: split 800 (±200) into 2 columns as 800 and 200)
- The ids used fro countries in the table don't match the ids of the API, but we can get the good ids by the country code
- The values in the table for total organic carbon stored don't match the values in the platform, we will use the values calculated in the platform.

Let us know when you have some feedback. Thanks!

**Comment by lyubovpencheva** [comment]
This is an old one.
